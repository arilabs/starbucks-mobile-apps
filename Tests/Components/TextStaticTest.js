import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import TextStatic from '../../app/Components/TextStatic/TextStatic.component';

test('TextStatic component renders correctly if text prop no set', () => {
  const tree = renderer.create(<TextStatic />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('TextStatic component renders correctly if text prop is set', () => {
  const tree = renderer.create(<TextStatic text="Hello world" />).toJSON();
  expect(tree).toMatchSnapshot();
});

test('TextStatic component renders correctly if text and styleText is set', () => {
  const tree = renderer.create(
    <TextStatic
      text="Hello world"
      styleText={{
        color: '#fff'
      }}
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
