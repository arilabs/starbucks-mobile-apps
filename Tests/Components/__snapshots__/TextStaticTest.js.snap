// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextStatic component renders correctly if text and styleText is set 1`] = `
<Text
  style={
    Object {
      "color": "#fff",
    }
  }
>
  Hello world
</Text>
`;

exports[`TextStatic component renders correctly if text prop is set 1`] = `
<Text>
  Hello world
</Text>
`;

exports[`TextStatic component renders correctly if text prop no set 1`] = `<Text />`;
